pages/examples/strip-plot.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import * as Plot from '@observablehq/plot';
+import Head from 'next/head';
+
+import { readData } from '../../helpers';
+
+enum AgeRange {
+  "<10",
+  "10-19",
+  "20-29",
+  "30-39",
+  "40-49",
+  "50-59",
+  "60-69",
+  "70-79",
+  "≥80",
+}
+
+const ages = Object.values(AgeRange).filter(x => typeof x === 'string');
+
+interface StateDistribution {
+  name: string,
+  "<10": number,
+  "10-19": number,
+  "20-29": number,
+  "30-39": number,
+  "40-49": number,
+  "50-59": number,
+  "60-69": number,
+  "70-79": number,
+  "≥80": number,
+}
+
+interface StateAge {
+  state: string,
+  age: AgeRange,
+  population: number,
+}
+
+interface Props {
+  states: StateDistribution[],
+  stateAges: StateAge[],
+}
+
+const Histogram: React.FC<Props> = ({ stateAges }) => {
```

Comment:
Let's make sure we change this name to reflect the right example!

```suggestion
const StripPlot: React.FC<Props> = ({ stateAges }) => {
```